python/ray/train/torch/torch_checkpoint.py

```diff
@@ -45,13 +88,24 @@ def from_model(
             >>> model = torch.nn.Identity()
             >>> checkpoint = TorchCheckpoint.from_model(model)
 
-            You can use a :py:class:`TorchCheckpoint` to create an
-            :py:class:`~ray.train.torch.TorchPredictor` and preform inference.
+            You can use a :class:`TorchCheckpoint` to create an
+            :class:`~ray.train.torch.TorchPredictor` and perform inference.
 
             >>> from ray.train.torch import TorchPredictor
             >>>
             >>> predictor = TorchPredictor.from_checkpoint(checkpoint)
-        """
+        """  # noqa: E501
+        # NOTE: For context on this `ValueError`, see issue #27922.
+        if model.__module__ == "__main__":
```

Comment:
does this also work for jupyter notebooks?

Reply:
I think so, will test

Reply:
Yes, it does